CHANGELOG.md

```diff
@@ -11,10 +11,11 @@
 
 **Bug Fixes**
 
+- Fix when removing `collectionName` from user role [\#1471](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1471) [Breaking Change] ([marcosuma](https://github.com/marcosuma)).
```

Comment:
I think breaking change should appear first. Also recommend applying formatting to make it stand out visually. Another small tweak to make the structure of this item match the others.
```suggestion
- **Breaking Change**: Fixes an issue where removing `collectionName` from user role doesn't work [\#1471](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1471) ([marcosuma](https://github.com/marcosuma)).
```